datalad/local/wtf.py

```diff
@@ -138,9 +139,29 @@ def _describe_system():
                                   _t2s(pl.win32_ver())]).rstrip(),
         'max_path_length': get_max_path_length(getpwd()),
         'encoding': get_encoding_info(),
+        'filesystem': [_get_fs_type(p) for p in [os.getcwd(),
+                                                 tempfile.gettempdir(),
+                                                 str(Path.home())]]
     }
 
 
+def _get_fs_type(path):
+    try:
+        from psutil import disk_partitions
+        match = ""
+        fs = ""
+        for part in disk_partitions():
+            if path.startswith(part.mountpoint) and \
+                    len(match) < len(part.mountpoint):
+                fs = part.fstype
+                match = part.mountpoint
+    except Exception as exc:
+        ce = CapturedException(exc)
+        lgr.warning("Failed to get filesystem information: %s", ce)
+        fs = tuple()
```

Comment:
How about something like this instead?

```suggestion
        fs = "Install 'psutils' to get filesystem information"
```

Reply:
I'm fine with removing the warning. I initially implemented it to match the behavior when other system information queries caused an exception.

Reply:
But I'm unsure what you mean with "infuse the message in the result rendering"?

Reply:
> But I'm unsure what you mean with "infuse the message in the result rendering"?

The renderer could - upon encountering such an empty entry - print that message instead. That would leave the result record itself intact as a structure that says "I got nothing on this", while conveying the suggestion to users.

However, I haven't looked into other cases. If you say that's how it's handled elsewhere, too, then this PR may not be the place to introduce a different approach.

Reply:
Thx! I've pushed one attempt to implement what you suggest in 33036ddcb95ad

Reply:
```
(handbook) adina@muninn in ~/Documents
❱ datalad wtf -S system
# WTF
## system 
  - distribution: debian/11-updates/n/a
  - encoding: 
    - default: utf-8
    - filesystem: utf-8
    - locale.prefered: UTF-8
  - filesystem: 
    - CWD: 
      - path: /home/adina/Documents
      - type: ext4
    - HOME: 
      - path: /home/adina
      - type: ext4
    - TMP: 
      - path: /tmp
      - type: ext4
  - max_path_length: 277
  - name: Linux
  - release: 5.16.0-3-amd64
  - type: posix
  - version: #1 SMP PREEMPT Debian 5.16.11-1 (2022-02-25)
(handbook) adina@muninn in ~/Documents
❱ pip uninstall psutil
Found existing installation: psutil 5.9.0
Uninstalling psutil-5.9.0:
  Would remove:
    /home/adina/env/handbook/lib/python3.9/site-packages/psutil-5.9.0.dist-info/*
    /home/adina/env/handbook/lib/python3.9/site-packages/psutil/*
Proceed (y/n)? y 
  Successfully uninstalled psutil-5.9.0
(handbook) adina@muninn in ~/Documents
❱ datalad wtf -S system
# WTF
## system 
  - distribution: debian/11-updates/n/a
  - encoding: 
    - default: utf-8
    - filesystem: utf-8
    - locale.prefered: UTF-8
  - filesystem: 
    - CWD: 
      - path: /home/adina/Documents
      - type: 
    - HOME: 
      - path: /home/adina
      - type: 
    - TMP: 
      - path: /tmp
      - type: 
  - max_path_length: 277
  - name: Linux
  - release: 5.16.0-3-amd64
  - type: posix
  - version: #1 SMP PREEMPT Debian 5.16.11-1 (2022-02-25)
Hint: Install psutils to get file system type information
```

Reply:
May be not most elegant solution, but I'm fine with that. We'll need to iron out the system for results, rendering and hints anyway. This is easy to find later, works and seems unlikely to cause trouble due to the global var - so, all good from my POV.
Thanks!